### ISSUE TITLE
OBJECT takes a string/number not a ARRAY reference (0.18_02)

### ISSUE THREAD

**Issue by eserte** [FGasper/p5-CBOR-Free#3]
Running Makefile.PL issues a warning:
```
$ perl5.18.4 Makefile.PL
WARNING: OBJECT takes a string/number not a ARRAY reference.
         Please inform the author.
...
```
In fact, it fails later while linking:
```
cc  -shared -O2 -L/usr/local/lib -fstack-protector ARRAY(0x26785f8)  -o blib/arch/auto/CBOR/Free/Free.so        \
        \
  
/bin/sh: 1: Syntax error: "(" unexpected
```
EUMM 6.66 is installed here, if this matters.

**Comment by FGasper** [comment]
Ah thank you! I’ve been wondering what causes this.

**Comment by FGasper** [comment]
FIxed via d3d78d2.
